pkg/formation/controller.go

```diff
@@ -131,7 +131,7 @@ func (c *Controller) processLocationDescriptor(logger *zap.Logger, ld *form_v1.L
 		if !ok {
 			return false, false, nil, errors.Errorf("release config map is missing expected data key: '%s'", releaseConfigMapDataKey)
 		}
-		err = yaml.Unmarshal([]byte(releaseStr), &releaseData)
+		err = yaml.UnmarshalStrict([]byte(releaseStr), &releaseData)
```

Comment:
what if there is a new field introduced on Deployinator side, but the client is not re-generated yet?

Reply:
ah, it's just `map[string]interface{}` rather than a struct, should be fine